config/redirects.js

```diff
@@ -2066,6 +2066,10 @@ module.exports = [
     from: ['/sso/current/integrations/office-365', '/integrations/sso/office-365','/integrations/office-365'],
     to: '/integrations/sso-integrations/office-365'
   },
+  {
+    from: ['/integrations/office-365-custom-provisioning'],
+    to: '/integrations/sso-integrations/office-365-custom-provisioning'
```

Comment:
This doesn't exist. Looks like the from URL is the correct one.

Reply:
You are correct